src/main.ts

```diff
@@ -31,7 +38,7 @@ function buildNpmDependencies(): any {
 }
 
 const command: Command = {
-	description: 'test your application',
+	description: 'this command will implicitly build your application and then run tests against that build',
```

Comment:
@bryanforbes can you raise a PR for that then and we can review and land and I will merge into this branch?